### ISSUE TITLE
securestreaming fails on GoogleTV when clip.accelerated = true

### ISSUE THREAD

**Issue by foresto** [flowplayer/flash#48]
I'm testing flowplayer-flash 3.2.15 on a Logitech Revue GoogleTV device, with .mp4 files and rtmpe streams (using the securestream plugin and a secret key shared with the rtmp server).

With default settings, flowplayer only plays the audio portion of both stream types (.mp4 and rtmpe).  Setting clip.accelerated = true gets the video to appear for .mp4 files, but it makes the rtmp secure stream fail to play at all.  That is, while rtmp audio would play with clip.accelerated = false, the player would just endlessly display the spinning circle animation with clip.accelerated = true.

Flash player version: GTV 10,2,164,128

Just to make sure that the Flash player on this device is capable of rtmpe + secure streaming, I verified that JWPlayer plays the same stream just fine.

**Comment by danrossi** [comment]
Do you have a demo page.

**Comment by foresto** [comment]
Yes, here's a demo page:

http://testweb.stealthytel.com/flowplayer/

**Comment by danrossi** [comment]
I dont have access to a google tv I'm afraid

https://dl.dropbox.com/u/3394987/flowplayer.commercial-3.2.15.zip

can you try this build perhaps.

**Comment by danrossi** [comment]
So your issue is rtmpe on google tv. Any ideas what the flash version is on these ?

**Comment by foresto** [comment]
Yes, as I wrote in my original post, Flash Player reports this version: GTV 10,2,164,128

**Comment by foresto** [comment]
I'm not sure how to try the build you linked above, since it doesn't have the secure token required for securestreaming to work. If you send me the source code for that build, I can build it with the appropriate token and give it a try.

**Comment by danrossi** [comment]
Hi there is no securestreaming module being used you are trying to use rtmpe which is an encrypted stream over stage video. This build has stage video fixes.

**Comment by foresto** [comment]
The securestreaming module is indeed being used. Note the code in my demo page that says this:

```
    playerconfig.clip.connectionProvider = 'secure';
```

Just to humor you, I tried the build you sent. It failed with TypeError: Error #1009.

If you have an RTMPE stream that does not use secure streaming, I'd be happy to test with that one instead, but I don't have one available to test with.

**Comment by danrossi** [comment]
Hi i dont see any securestreaming plugin loaded, if you have built it in you will need to update the sources from this branch for now https://github.com/flowplayer/flash/tree/danielr-httpstreaming. I would also rebuild the controlbar etc.

**Comment by foresto** [comment]
I did indeed build it in.  I'll give that branch a try; thanks.

**Comment by danrossi** [comment]
Yes it sounds like my stage video fixes might work for you. It should be released soon.

**Comment by foresto** [comment]
Okay, I tried your branch (cloned with git a few minutes ago).  The problem isn't fixed, but I did notice that if I keep retrying (by repeatedly clicking my demo page button that starts the stream) the video does sometimes play.  I'd say it works about 20% of the time.

**Comment by danrossi** [comment]
sorry for the late reply, what version of android is it running and what browser ? I dont have google tv, but testing on android is possible.
